web/src/stories/stepper.stories.mdx

```diff
@@ -115,3 +140,160 @@ export const StepperStoryTemplate = args => ({
 
 <h2>Props</h2>
 <Props story = "Stepper component"/>
+
+export const stepperHFDocs = {
+  source: {code: `title = "Example Stepper with Horizontal Form" component.NewStepper()`},
```

Comment:
Add a snippet of working code that reflects what is shown

Reply:
done!